pysqldb3/README.md

```diff
@@ -1 +1,88 @@
-# pysqldb3
\ No newline at end of file
+## Pysqldb3 Library Components 
+
+## 1. Pysqldb3 (DbConnect) - Pysqldb3.py
+Pysqldb contains DbConnect, a class built off of pymssql/psycopg2 that makes it easy to connect to Postgres/Sql Server databases in Python. 
+### 1.1 Simple Example/Setup  
+To create a DbConnect object:
+1. Import 
+   `from pysqldb3 import pysqldb3`
+2. `db = pysqldb.DbConnect(user=None, password=None, ldap=False, type=None, server=None, database=None, port=5432,
+                 allow_temp_tables=False, use_native_driver=True, default=False, quiet=False))` 
+    + Replace server_name and db_name with the actual values. 
+    + You must specify the type, either `PG` (Postgres) or `MS` (SQL Server). 
+    + You can also add your username using `user='my_username'` 
+    + You can add your password similarly using `password='mypassword'`
+    + Other inputs include LDAP and ports. If you're unsure about these, feel free to ask around. 
+3. Now, you can query on your database by using `db.query('select * from my_schema.my_table')`
+
+### 1.2 Functions 
+In Jupyter or Python shell, use help(pysqldb) to show all public functions and their inputs. <br>
+#### pysqldb3 public functions:
+
+
+1. [`DbConnect`](#connect): Connects to database
+1. [`disconnect`](#disconnect): Disconnects from database.  
+1. [`check_conn`](#check_conn): Checks and reconnects to connection if not currently connected.
+
+### DbConnect
+**`pysqldb3.DbConnect.(quiet=False)`**
+Creates database connection instance.  
+###### Parameters:
+ - **`quiet`: bool, default False**: When true, does not print database connection information.
+ 
+ 
+**Sample** 
```

Comment:
this sample doesnt show the `db.connect`, its showing `db.DbConnect` call

Reply:
ok, will show db.connect